src/pages/_app.js

```diff
@@ -100,10 +100,10 @@ function App({ Component, pageProps }) {
 
   return (
     <>
-      <Head>
-        <title>My page</title>
-        <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width" />
-      </Head>
+      <Head
+        title="Your Story Matters"
+        ogImage="logo.png"
+      />
```

Comment:
If `Head` is also declared in the app template (as well as individual pages), won't the tags be duplicated? (unless I'm missing something here).

Reply:
Next/Head overwrites them, so it gives us a default whilst we're waiting for content to load etc...